<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Akshays-MacBook-Air.local" failures="1" tests="3" name="Test" time="0.201" errors="0" timestamp="2023-10-07T22:21:34 IST">
  <testcase classname="com.testscripts.Runner" name="practiseMultipleMethodsinSelenium" time="0.084">
    <skipped/>
  </testcase> <!-- practiseMultipleMethodsinSelenium -->
  <testcase classname="com.testscripts.Runner" name="practiseMultipleMethodsinSelenium" time="0.007">
    <skipped/>
  </testcase> <!-- practiseMultipleMethodsinSelenium -->
  <testcase classname="com.testscripts.Runner" name="practiseMultipleMethodsinSelenium" time="0.007">
    <failure type="org.apache.poi.openxml4j.exceptions.NotOfficeXmlFileException" message="No valid entries or contents found, this is not a valid OOXML (Office Open XML) file">
      <![CDATA[org.apache.poi.openxml4j.exceptions.NotOfficeXmlFileException: No valid entries or contents found, this is not a valid OOXML (Office Open XML) file
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.getNextEntry(ZipArchiveThresholdInputStream.java:156)
at org.apache.poi.openxml4j.util.ZipInputStreamZipEntrySource.<init>(ZipInputStreamZipEntrySource.java:94)
at org.apache.poi.openxml4j.opc.ZipPackage.<init>(ZipPackage.java:132)
at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:319)
at org.apache.poi.ooxml.util.PackageHelper.open(PackageHelper.java:59)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:290)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:286)
at core.utils.ReadExcel2.getSheet(ReadExcel2.java:26)
at core.utils.ReadExcel2.getExcelAsMap(ReadExcel2.java:32)
at core.utils.ReadExcel2Test.readExcelSheet(ReadExcel2Test.java:20)
at core.utils.InitiateBrowser.initialiseBrowserAndPassData(InitiateBrowser.java:29)
at com.testscripts.Runner.practiseMultipleMethodsinSelenium(Runner.java:19)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.zip.ZipException: Unexpected record signature: 0X474E5089
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.getNextZipEntry(ZipArchiveInputStream.java:296)
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.getNextEntry(ZipArchiveThresholdInputStream.java:152)
... 26 more
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- practiseMultipleMethodsinSelenium -->
</testsuite> <!-- Test -->
